src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1532,6 +1532,16 @@ Notifications will only be sent for jobs whose generic information contains user
 </genericInformation>
 ----
 
+=== Chaining Jobs
+
+The SubmitJobNoWait and SubmitJobAndWait templates under the *Controls* menu allow you to submit a workflow from another workflow through using *schedulerapi*.
+
+In order to submit a workflow, you need to put the URL of that workflow in Task Variables under the key *workflowURL*. You can use whatever URL, if that URL is
```

Comment:
.. that workflow in the Task Variables.....
...You can use any URL that points to a valid workflow xml file.